pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
What do we need this file for?

Reply:
Running "pytest" in the root directory runs all files with "test" in their name. There's an example `pandas_train_test_example.py" that get's run along with them that takes a while and generates a plot. This file just specifically tells pytest that it should look in the "test" directory.